components/ModalManager/InterestZoneModalHeader.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { ActionIcon, Button, Group, MediaQuery, Select, Text, Title } from '@mantine/core';
+import { InterestZone, Status } from '../../models/zone.model';
+import { ModalConfig } from '../Providers/ModalProvider';
+import { ArrowBarLeft, BrandStackoverflow, Edit, Trash } from 'tabler-icons-react';
+
+export default function ZoneModalHeader({
+  modal,
+  zone,
+  setModal,
+}: {
+  modal: ModalConfig;
+  zone: InterestZone | Partial<InterestZone>;
+  setModal: (modal: ModalConfig | undefined) => void;
+}) {
+  const statusColor: Record<Status, string> = {
+    [Status.DONE]: '#B0EF8F',
```

Comment:
some constants need to be defined instead of using the colorcodes directly (for reusability)